### ISSUE TITLE
`select` is not accessible for keyboard navigation

### ISSUE THREAD

**Issue by Guria** [bem-archive/bem-site-engine#208]
`select` in `bem-components` examples works fine and allows to open/close popup and select item from keyboard.
But version/level selectors in library section and sorting setting in forum section only able to open popup, but item can be selected only with mouse.

**Comment by Guria** [comment]
There is one more issue with this select:
- click into select (popup opened)
- click into select again (popup closed)
- try to press spacebar (popup opened and closed immediately)
